src/common/config/state/init.js

```diff
@@ -9,6 +9,7 @@ const model = () => {
   }
   return {
     allow_bfcache: true, // *cli - temporary feature flag for BFCache work
+    assetsPath: undefined,
```

Comment:
can we just add a little comment to this for future sanity
```suggestion
    assetsPath: undefined, // if a value is set here by a customer, it will be used to overwrite the webpack asset path used to fetch assets
```